### PR TITLE
Fixed #28820 -- Eliminated an extra query with QuerySet.update() on proxy models.

### PR DESCRIPTION
**Pull request by vonafor**
Ticket: https://code.djangoproject.com/ticket/28820

### CODE REVIEW

File: tests/model_inheritance/tests.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def setUpTestData(cls):
             rating=2,
         )
 
-        chef = Chef.objects.create(name="Albert")
+        chef = Chef.objects.create(name="Albert", id=1)
```
Comment (charettes) [inline tests/model_inheritance/tests.py]: Assign `chef` to `cls` and use `self.chef.id` below instead as hardcoding pks might cause some issues.

File: tests/model_inheritance/tests.py [hunk 2]
```diff
@@ -331,6 +331,13 @@ def test_update_works_on_parent_and_child_models_at_once(self):
         self.assertFalse(r1.serves_hot_dogs)
         self.assertEqual(r1.name, "Demon Puppies")
 
+    def test_update_query_counts_for_proxy(self):
+        """
+        #28820
```
Comment (timgraham) [inline tests/model_inheritance/tests.py]: Docstrings should state the expected behavior. Ticket references should be reserved for obscure issues. These guidelines are from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

File: tests/model_inheritance/tests.py (hunk 1 above)
Comment (timgraham) [reply tests/model_inheritance/tests.py]: Better to put the test in `proxy_model_inheritance` so a new model isn't needed.

**Comment by vonafor** [comment]
Thank you for review. I've updated my changes.

**Pull request merged**